### DISCUSSION TITLE
忽然想到一个问题

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#891] [Q&A]
在 [这里](https://github.com/Arondight/Adachi-BOT/tree/master/resources/Version2/info/docs) 只有 [旅行者](https://github.com/Arondight/Adachi-BOT/blob/master/resources/Version2/info/docs/%E6%97%85%E8%A1%8C%E8%80%85.json) 的一个 `json` 文件，

但实际上，旅行者是可以替换元素的，所以是否应该加上诸如  `岩旅行者`、 `雷旅行者`、 `草旅行者` 之类的指向特定元素旅行者的 `json`

~~算上还可以改变性别那就要再加上`草男旅行者`、`雷女旅行者`之类的`json`~~

**Comment by mark9804** [comment]
很早之前讨论过这个问题，我们觉得旅行者有一个风元素的意思意思就行了 ~~不要不识抬举~~

**Comment by BTMuli** [comment]
但这个确实有需求啊（而且也不麻烦

**Reply by mark9804** [reply]
对于自动工作流来说是有点麻烦的，因为旅行者有重名，需要额外的逻辑去处理这种情况

**Reply by BTMuli** [reply]
用`信息`查询的话没有特意指出就按现在默认的来，只是加上`雷旅行者.json`应该不会混淆吧

**Reply by mark9804** [reply]
emmm，当时我们不加主要是因为没有那个时间和经历，因为旅行者的信息现在只能手动录入（ID重复了），要是你有信心坚持到至冬的话这个艰巨的任务也许可以你来 PR

**Reply by BTMuli** [reply]
我得看看 [`api.js`](https://github.com/Arondight/Adachi-BOT/blob/master/src/utils/api.js) 以及其他的一些文件，了解`信息 info` 是如何实现的再进行改动

**Reply by mark9804** [reply]
按名字找的

**Comment by Arondight** [comment]
以后做好图片自动化之后，旅行者的信息我来搞，但是只区分元素，不区分性别，一律用老妹

**Comment by mark9804** [comment]
感觉如果 #943 做完之后可能顺带着这个功能就被实现掉了
